crypto/ui/ui_openssl.c

```diff
@@ -190,8 +190,8 @@ static int read_string(UI *ui, UI_STRING *uis);
 static int write_string(UI *ui, UI_STRING *uis);
 
 static int open_console(UI *ui);
-static int echo_console(UI *ui);
 static int noecho_console(UI *ui);
+static int echo_console(UI *ui);
```

Comment:
While you are at it, maybe move echo_console back, so that there are no unnecessary changes... Note "maybe", as I don't actually insist...